### PR TITLE
Added some options

### PR DESCRIPTION
**Pull request by andras-tim**
* flake8-max-complexity
 * flake8-show-source
 * flake8-statistics

### CODE REVIEW

File: pytest_flake8.py [hunk 1]
```diff
@@ -29,6 +29,15 @@ def pytest_addoption(parser):
         "flake8-max-line-length",
         help="maximum line length")
     parser.addini(
+        "flake8-max-complexity",
+        help="McCabe complexity threshold")
+    parser.addini(
```
Comment (tholo) [inline pytest_flake8.py]: Should add a `type='bool'` argument here...

File: pytest_flake8.py [hunk 2]
```diff
@@ -29,6 +29,15 @@ def pytest_addoption(parser):
...
+        "flake8-show-source",
+        help="show the source generate each error or warning")
+    parser.addini(
```
Comment (tholo) [inline pytest_flake8.py]: And here

File: pytest_flake8.py [hunk 3]
```diff
@@ -38,6 +47,9 @@ def pytest_configure(config):
     if config.option.flake8:
         config._flake8ignore = Ignorer(config.getini("flake8-ignore"))
         config._flake8maxlen = config.getini("flake8-max-line-length")
+        config._flake8maxcomplexity = config.getini("flake8-max-complexity")
+        config._flake8showshource = config.getini("flake8-show-source").upper() == 'TRUE'
```
Comment (tholo) [inline pytest_flake8.py]: Then these will return bool values directly

File: pytest_flake8.py [hunk 4]
```diff
@@ -38,6 +47,9 @@ def pytest_configure(config):
...
+        config._flake8statistics = config.getini("flake8-statistics").upper() == 'TRUE'
```
Comment (tholo) [inline pytest_flake8.py]: Here too

**Comment by tholo** [comment]
Added a couple of comments, looks good otherwise.

**Comment by andras-tim** [comment]
Dear @tholo,
Sorry for my slooow reaction. I fixed the code according to your comments.

**Comment by tholo** [comment]
Cool, thank you!

**Comment by andras-tim** [comment]
Mee too.
When can you release the changes? https://github.com/tholo/pytest-flake8/compare/0.5...master

**Comment by tholo** [comment]
0.6 was already released, have just released 0.7 with the above changes

**Comment by andras-tim** [comment]
thank you!

**Pull request merged**